### ISSUE TITLE
Move install.sh into composer

### ISSUE THREAD

**Issue by leevigraham** [venveo/craft-boilerplate#1]
Hi guys,

Nice work on the bolierplate. This is more of a question than an issue. Is there any reason why you haven't moved the install.sh into composer.php scripts?

Something like:

``` json
{
    "name": "nsm/craft-boilerplate",
    "type": "project",
    "description": "Newism's Craft Boilerplate",
    "keywords": [
        "craft", "craftcms", "boilerplate", "craftboilerplate", "craft boilerplate"
    ],
    "authors": [
        {
            "name": "Leevi Graham",
            "email": "leevi@newism.com.au"
        }
    ],
    "support": {
        "issues": "https://github.com/newism/craft-boilerplate/issues",
        "source": "https://github.com/newism/craft-boilerplate"
    },
    "scripts": {
        "post-root-package-install": [],
        "post-create-project-cmd": [
            "@composer craft:install",
            "cp .env.example .env"
        ],
        "craft:install": [
            "mkdir -p craft",
            "curl -kLo ./craft.zip 'https://craftcms.com/latest.zip?accept_license=yes'",
            "unzip craft.zip -d tmp",
            "mv tmp/craft/app craft/app",
            "rm -rf tmp && rm craft.zip"
        ]
    },
    "require": {},
    "require-dev": {
        "vlucas/phpdotenv": "^2.4"
    }
}

```

**Comment by themccallister** [comment]
@leevigraham thank you for the feedback! 

This is honestly something we considered or even looked into and honestly it makes perfect sense. I do have a few changes I need to tweak based on some development environments, this will be one of them, and then tag a new release.

I will look into moving this into master as soon as I have a few free moments. I will close this out when it is merged!

**Comment by leevigraham** [comment]
@themccallister No worries.

There's always more than one way to skin a cat. In the case of packages + scripts there's 1000's. I think it makes the most sense to keep everything in composer given CraftCMS is a php package and you've created a Craft project installer.

Nice work on the boilerplate.

**Comment by themccallister** [comment]
Apparently tagging releases does not close an issue automatically. Closing this as it was changed in #2 and merged into [v1.2.0](https://github.com/venveo/craft-boilerplate/releases/tag/v1.2.0).

Thanks again @leevigraham!
